### DISCUSSION TITLE
I don't know why the 691 version, works fine, and 694 directly turns into an error where you fail to load properly

### DISCUSSION THREAD

**Discussion by sabergg5** [ldtteam/minecolonies#10352] [General]
I don't know why the 691 version, works fine, and 694 directly turns into an error where you fail to load properly

**Comment by sabergg5** [comment]
[Uploading 错误报告-2024-10-23_0.08.39.zip…]()

**Comment by sabergg5** [comment]
mod 400+

**Comment by Raycoms** [comment]
Please make an issue and upload your log to a gist.
